docs/Help.md

```diff
@@ -10,13 +10,66 @@ changed.  Also a good place to look for new goodies.
 
 ## Help topics
 
-* [Travis CI file explained](TravisFileExplained.md): every line of the `.travis.yml` file explained.
+* [Travis CI file explained](TravisFileExplained.md): every line of the `.travis.dist.yml` file explained.
+* [GitHub Actions workflow file explained](GHAFileExplained.md): every line of the `gha.dist.yml` file explained.
 * [Add extra Moodle configs](AddExtraConfig.md): how to add extra configs to Moodle `config.php`.
 * [Add extra plugins](AddExtraPlugins.md): how to add plugin dependencies to Moodle.
 * [Ignoring files](IgnoringFiles.md): how to ignore files that might be causing failures.
 * [Generating code coverage](CodeCoverage.md): how to generate code coverage of your plugin.
 * [CLI commands and options](CLI.md): the available `moodle-plugin-ci` commands and their options.
 
+## Test steps quick start
+
+The snippet below may help you to optimise set of test steps you want to
+inclide in the CI scenario.
+
+```
```

Comment:
Personal opinion 100%!

This is the most unreadable stone of text I've seen in my life, lol. Hope people goes to [CLI commands and options](https://moodlehq.github.io/moodle-plugin-ci/CLI.html) instead :-)

Note I understand it's a quick-reference but, at least for me, it's highly unreadable. No need to change it now (or maybe ever, ask for other opinions, please). But it hammered my brain! :-D

Reply:
Haha, it is from travis file docs, I didn't want to duplicate it in GHA workflow, but wanted to keep it somewhere outside as it applicable to all methods. I will change it to normal text if that will help, just was lazy.

Reply:
Ah, 100% up to you. If you think it's ok, no problem here!

Reply:
> I’ve seen in changelog (for the version to be released):
> 
> `“...See #46 for more details...”`
> 
> Does that work (converting to link) or do we need to provide the link to #46 ? Other cases in the changelog do have the complete URL to the issues/PRs.

You are right, it does not get converted to URL: https://github.com/moodlehq/moodle-plugin-ci/blob/master/docs/CHANGELOG.md

Reply:
Much better, heap thanks!